scripts/variantstore/variant-annotations-table/GvsValidateVAT.wdl

```diff
@@ -153,7 +157,7 @@ workflow GvsValidateVat {
     }
 
     # only check certain things if the callset is larger than 10,000 samples (a guess)
-    Boolean callset_is_small = GetNumSamplesLoaded.num_samples < 10000
+    Boolean callset_is_small = select_first([is_small_callset, select_first([GetNumSamplesLoaded.num_samples, 1]) < 10000])
```

Comment:
what does this do?

Reply:
maybe a comment would be helpful or am I just WDL illiterate?

Reply:
I'm adding a comment. Basically just checking if the `is_small_callset` flag is set, if not use the previous logic (where it's considered a small callset if there are less than 10000 samples).